operations/kubernetes/Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md

```diff
@@ -181,6 +181,6 @@ Data is repopulated in BSS when the REDS `init` job is run.
         **`NOTE`** Modify the `-w` arguments in the following commands to reflect the number of worker and storage nodes in the system.
 
         ```bash
-        pdsh -w ncn-w00[0-4]-can.local "systemctl restart cray-dvs-orca"
-        pdsh -w ncn-s00[0-4]-can.local "systemctl restart cray-dvs-orca"
+        pdsh -w ncn-w00[0-4]-can.local "systemctl restart cray-orca"
+        pdsh -w ncn-s00[0-4]-can.local "systemctl restart cray-orca"
```

Comment:
Why are these `pdsh` commands against the `can` interface on the NCNs?

Also this step and the one before are missing the command prompt. https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts-inside-of-directions

Reply:
Added command prompt.  Not sure why it uses the can bus.  Appears to work.  Adjusted range to be 1-4 instead of 0-4